consts.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+# Manually update this after each minor CMS release
+const INSTALLER_TO_PHP_VERSIONS = [
+    '4.9' => [
+        '7.1',
+        '7.2',
+        '7.3',
+        '7.4'
+    ],
+    '4.10' => [
+        '7.3',
+        '7.4',
+        '8.0',
+    ],
+    '4.11' => [
+        '7.4',
+        '8.0',
+        '8.1',
+    ],
+    '4' => [
+        '7.4',
+        '8.0',
+        '8.1',
+    ],
+];
+
+const DB_MYSQL_57 = 'mysql57';
+const DB_MYSQL_57_PDO = 'mysql57pdo';
+const DB_MYSQL_80 = 'mysql80';
+const DB_PGSQL = 'pgsql';
+
+// Used when determining the version of installer to used. Intentionally doesn't include recipes
+const LOCKSTEPED_REPOS = [
+    'silverstripe-admin',
+    'silverstripe-asset-admin',
+    'silverstripe-assets',
+    'silverstripe-campaign-admin',
+    'silverstripe-cms',
+    'silverstripe-errorpage',
+    'silverstripe-framework',
+    'silverstripe-reports',
+    'silverstripe-siteconfig',
+    'silverstripe-versioned',
+    'silverstripe-versioned-admin',
+    // recipe-solr-search is not a true recipe in that i cannot be used to create a project
+    // i.e. it doesn't include recipe-cms or recipe-core unlike our other recipes
```

Comment:
I think the long comment is unnecessary - the definition of a "true recipe" is moot, all that matters here is that it doesn't include one of `recipe-cms` or `recipe-core`

My original suggestion below illustrates that point without the extra unrelated information.
> recipe-solr-search doesn't include recipe-cms or recipe-core unlike our other recipes

Reply:
Updated